src/screens/Shop/Shop.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Screen from '../../styledComponents/Screen'
+import Filters from '../../components/Filters/Filters'
+import ProductsList from '../../components/ProductsList/ProductsList'
+
+const Shop = () => {
+    return (
+      <Screen>
+        <Filters/>
+        <ProductsList/>
```

Comment:
Is this a good practice? Importing from file is ok, but we should at least pretend that some kind of server is the source